dashboard/app/models/census/ap_cs_offering.rb

```diff
@@ -54,6 +54,10 @@ def self.seed_from_csv(course, school_year, filename)
 
   CENSUS_BUCKET_NAME = "cdo-census".freeze
 
+  def self.construct_object_key(course, school_year)
+    "ap_cs_offerings/#{course}-#{school_year}-#{school_year + 1}.csv"
```

Comment:
This isn't changing here, just moving, but out of curiosity why `school_year + 1`?

Reply:
School years span calendar years so they will be, for example, 2017-2018. When we're loading data for all school years we need to iterate through them via `(2017..current_year).each do |school_year|` and then translate that into the string with the start and end year.